.github/workflows/test-suite.yml

```diff
@@ -9,19 +9,20 @@ on:
 
 jobs:
   tests:
-    name: "Python ${{ matrix.python-version }}"
-    runs-on: "ubuntu-latest"
-
+    name: "Python ${{ matrix.python-version }} ${{ matrix.os }}"
+    runs-on: "${{ matrix.os }}"
     strategy:
       matrix:
         python-version: ["3.6", "3.7", "3.8"]
-
+        os: [windows-latest, ubuntu-latest]
     steps:
       - uses: "actions/checkout@v2"
       - uses: "actions/setup-python@v1"
         with:
           python-version: "${{ matrix.python-version }}"
       - name: "Install dependencies"
         run: "scripts/install"
+        shell: bash
```

Comment:
Do we need to be specific here because of the windows case, or is this an unrelated bit of tidying up?

Reply:
yes, according to https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell the default shell for windows os would be powershell and we would have to write a ps1 script, that's what I did before @tianglo pointed me into that better direction

> powershell | This is the default shell used on Windows. The Desktop PowerShell. GitHub appends the extension .ps1 to your script name.
-- | --